common/src/stack/examples/spreadsheets/partition/redhat-uefi.csv

```diff
@@ -0,0 +1,7 @@
+NAME,DEVICE,MOUNTPOINT,SIZE,TYPE,OPTIONS
+backend-0-0,sda,/boot/efi,512,efi,
+,sda,swap,8192,swap,
+,sda,/export,15360,xfs,
+,sda,/boot,1024,xfs,
```

Comment:
Why give Redhat a separate boot partition when `/boot/efi` is already separate? Just another small partition to fill up with updated kernels (which sucks, speaking from past experience).

Reply:
My thinking was to keep it similar to the other partition schemes with a separate /boot partition. I think the reasoning there is in case root gets messed up (`/boot/efi` only holds the initial .efi files for booting not the initrd or other files) or for wider distro compatibility (for more info I found this: https://superuser.com/questions/522971/is-a-boot-partition-always-necessary).